src/template.rs

```diff
@@ -147,8 +159,28 @@ impl<'a> Renderer<'a> {
             self.handlebars
                 .register_templates_directory(".hbs", templates)?;
         }
-        self.handlebars
-            .register_templates_directory(".hbs", &self.template_dir)?;
+        for entry in WalkDir::new(&self.template_dir)
+            .into_iter()
+            .filter_map(|e| e.ok())
+        {
+            if entry.file_type().is_file() && entry.path().extension().map_or(false, |e| e == "hbs")
+            {
+                let filename = entry.file_name().to_str().unwrap().replace(".hbs", "");
+                let res = parse_hbs_template(entry);
+                match res {
+                    Ok((template_meta, body)) => {
+                        self.template_meta
+                            .insert(filename.to_owned(), template_meta);
+                        self.handlebars.register_template_string(&filename, body)?;
+                    }
+                    Err(err) => {
+                        eprintln!("Error reading template {}: {}", filename, err);
+                    }
+                }
+            }
+        }
+        // self.handlebars
```

Comment:
Is this commented-out code safe to remove?

Reply:
yep